iset.mm

```diff
@@ -69304,18 +69330,36 @@ property of disjoint unions (see ~ updjud in the case of functions).
       A. i e. _om ( f ` suc i ) C_ ( f ` i ) } $.
   $}
 
+  ${
+    $d f i j $.
+    $( The point at infinity in ` NN+oo ` is the constant sequence equal to
+       ` 1o ` .  Note that with our encoding of functions, that constant
+       function can also be written as ` ( _om X. { 1o } ) ` , as ~ fconstmpt
+       shows.  (Contributed by Jim Kingdon, 14-Jul-2022.)  Use maps-to
+       notation.  (Revised by BJ, 10-Aug-2024.) $)
+    infnninf $p |- ( i e. _om |-> 1o ) e. NN+oo $=
```

Comment:
I think currently we have a mix of ` ( i e. _om |-> 1o ) ` and ` ( _om X. { 1o } ) `; I'm fine with standardizing on ` ( i e. _om |-> 1o ) `.

Reply:
Yes, I think it is better when we really mean the **function** and not a particular **set**. Indeed, that the latter set represents the former constant function is an "implementation detail" (even though a standard one in treatments based on set theory).

Reply:
If you are reading set.mm or iset.mm in order, you run into sets of ordered pairs before you run into the maps-to notation, but it is a fair point that in some foundations, functions are fundamental rather than built on sets, so :shrug: . Once we have introduced the maps-to notation, might as well use it for constant functions as well as others.

Reply:
It is also the case that functions may be defined slightly differently even in a ZF-based foundation: there are treatments (and it is the one I was taught as a freshman), where a function "is" a triple (A, X, B) where $X \subseteq A \times B$ is such that ... That is, the domain and codomain are explicitly given as A and B. In these foundations, the function `( i e. _om |-> 1o )` would be `<. _om , ( _om X. { 1o } ) , 2o >.` (and the codomain 2o should somehow appear in the maps-to notation too). That encoding has the merit of specifying the codomain, which is not the case with our encoding.

Reply:
The maps-to notation would need to be modified in that case to specify the codomain. This is mentioned at https://en.wikipedia.org/wiki/Function_(mathematics)#Arrow_notation

Anyway, this is getting pretty far afield from our current issue given that we both agree on what to do.

Reply:
Yes, that's what I meant above by my "should somehow appear".

No worries: the point of this exchange is not to convince one way is better than the other: just to exchange information.